src/com/core/tictactoe/GamesController.java

```diff
@@ -1,36 +1,29 @@
 package com.core.tictactoe;
 
-import com.core.tictactoe.game_options.ExitGameOption;
 import com.core.tictactoe.game_options.GameOption;
 import com.core.tictactoe.game_options.GameOptionsFactory;
-import com.core.tictactoe.game_options.RunGameOption;
 
 public class GamesController {
 
     private CommandLineUI commandLineUI;
-    private String gameStatus;
-    public Player firstPlayer;
+    private Boolean runProgram;
 
     public GamesController(CommandLineUI commandLineUI) {
         this.commandLineUI = commandLineUI;
-    }
-
-    public String getGameStatus() {
-        return this.gameStatus;
+        this.runProgram = true;
     }
 
     public void run() {
-        while (true) {
+        while (this.runProgram) {
             this.commandLineUI.gamingMenu();
             String userInput = this.commandLineUI.getUserInput();
             userInput = this.commandLineUI.askWhoGoesFirst(userInput);
             GameOption gameOption = GameOptionsFactory.get(userInput);
-            gameOption.run(this.commandLineUI);
-            if (gameOption instanceof ExitGameOption) break;
-            if (gameOption instanceof RunGameOption) {
-                this.gameStatus = "played";
-                this.firstPlayer = ((RunGameOption) gameOption).getPlayerOne();
-            }
+            this.runProgram = gameOption.run(this.commandLineUI);
         }
     }
+
+    public boolean getRunProgramStatus() {
```

Comment:
It looks like this method is only being used in your tests – 🚩
It would be good to find a way to test the behaviour without exposing extra public methods that aren't needed by other classes.